stac-example/src/main/scala/geotrellis/stac/raster/StacCollectionSource.scala

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2019 Azavea
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package geotrellis.stac.raster
+
+import io.circe.syntax._
+
+import geotrellis.stac._
+import geotrellis.raster._
+import geotrellis.vector._
+import com.azavea.stac4s.StacCollection
+
+case class StacCollectionSource(asset: StacCollection, source: RasterSource) extends StacSource[StacCollection] {
+  val extent: Extent   = asset.extent.spatial.toExtent
+  val name: SourceName = StringName(asset.id)
+
+  lazy val attributes: Map[String, String] =
```

Comment:
Is this value provided in a few places for OGC attribute serialization? I'm confused about what it's for

Reply:
@jisantuc that is a stab for some metadata, done for compat reasons with RasterSources, but I'll double check it. A good point.

Reply:
Compat with OGCSources https://github.com/geotrellis/geotrellis-server/pull/345/files#diff-aefa2eec08a0612f876442b002652b0dde1ef21db90c8616f897adaa9dfce2d8R47